ospfd/ospf_api.c

```diff
@@ -378,8 +378,12 @@ struct msg *msg_read(int fd)
 
 	/* Determine body length. */
 	bodylen = ntohs(hdr.msglen);
-	if (bodylen > 0) {
+	if (bodylen > (ssize_t)sizeof(buf)) {
+		zlog_warn("msg_read: Body Length of message greater than what we can read");
```

Comment:
can we use `__func__` here instead of static?

Reply:
fixed